sysboundary/ionosphere.cpp

```diff
@@ -67,6 +67,7 @@ namespace SBC {
          Readparameters::add(pop + "_ionosphere.VX0", "Bulk velocity of ionospheric distribution function in X direction (m/s)", 0.0);
          Readparameters::add(pop + "_ionosphere.VY0", "Bulk velocity of ionospheric distribution function in X direction (m/s)", 0.0);
          Readparameters::add(pop + "_ionosphere.VZ0", "Bulk velocity of ionospheric distribution function in X direction (m/s)", 0.0);
+         Readparameters::add(pop + "_ionosphere.fluffiness", "Weight of boundary (0) vs. average of NOT_SYSBOUNDARY neighbor's (1) moments and velocity distribution.", 0);
```

Comment:
The uninitiated would have no idea what to make of that help text. How about
"Inertia of boundary smoothing when copying neighbour's moments and velocity distributions (0=completely constant boundaries, 1=neighbours are interpolated immediately"

Reply:
Pasted that in.